containers/datalab/Dockerfile.in

```diff
@@ -115,6 +118,16 @@ RUN cd /datalab/lib/GCPData-0.1.0 && python setup.py install && \
     cd /datalab/web && /tools/node/bin/npm install && \
     cd /
 
+# Link stuff needed for nbconvert to a location where Jinja will find it.
```

Comment:
I could just fold this in with the previous RUN using && rather than adding a script. Thoughts?